### PR TITLE
Added describe for tests

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

### CODE REVIEW

File: extensions/logs/logs.go [hunk 1]
```diff
@@ -200,12 +200,12 @@ func capture(name string) context.CancelFunc {
...
+	}, dir
 }
 
 // Capture returns a function that saves logs since Capture function has been called.
-func Capture(name string) context.CancelFunc {
-	c := capture(name)
+func Capture(name string) (captureFunc context.CancelFunc, dir string) {
+	c, dir := capture(name)
 
 	return func() {
```
Comment (denis-tingaikin) [inline extensions/logs/logs.go:210]: please add describe here.

Comment (denis-tingaikin) [inline extensions/logs/logs.go:210]:
```suggestion
	return func() {
	   storeDescribes()
```

Comment (Mixaster995) [reply extensions/logs/logs.go:210]: ok, fixed

Comment (Mixaster995) [reply extensions/logs/logs.go:210]: ok, done

File: extensions/logs/logs.go [hunk 2]
```diff
@@ -200,14 +200,16 @@ func capture(name string) context.CancelFunc {
...
 		captureLogs(now, dir)
-	}
+	}, dir
 }
 
 // Capture returns a function that saves logs since Capture function has been called.
-func Capture(name string) context.CancelFunc {
-	c := capture(name)
+func Capture(name string, runner func(cmd string)) context.CancelFunc {
```
Comment (denis-tingaikin) [inline extensions/logs/logs.go:207]:
```suggestion
func Capture(name string) context.CancelFunc {
```

File: extensions/logs/logs.go [hunk 3]
```diff
@@ -200,14 +200,16 @@ func capture(name string) context.CancelFunc {
...
+	c, dir := capture(name)
 
 	return func() {
+		runner("kubectl describe pods -n nsm-system >" + filepath.Join(dir, "describe"))
```
Comment (denis-tingaikin) [inline extensions/logs/logs.go:211]:
```suggestion
                 var runner shell.Bash
		runner.Run("kubectl describe pods -n nsm-system >" + filepath.Join(dir, "describe"))
```

Comment (Mixaster995) [reply extensions/logs/logs.go]: ok, done

File: extensions/logs/logs.go (hunk 2 above)
Comment (Mixaster995) [reply extensions/logs/logs.go]: ok, done

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**